student.rb

```diff
@@ -10,5 +10,5 @@ def play_hooky
     '¯(ツ)/¯'
   end
 end
-student = Student.new('A class ', 18, 'Ali', parent_permission: false)
-puts student.id
+# student = Student.new('A class ', 18, 'Ali', parent_permission: false)
+# puts student.id
```

Comment:
- Can you remove commented codes? Because it creates confusion and also makes codes look dirty.